Bottom: e11d60006ff12ec471d11ff132f227a73fb4b41e
Top:    e3a70a44588ccbcfc7d84b80cc31fba88502dbff
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-09-15 16:26:03 +0800

MAINTAINERS: Stefan will not maintain network device layer

Talked with Stefan, he will not maintain network device layer.

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/MAINTAINERS b/MAINTAINERS
index 688979b..2f4e8cf 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -955,11 +955,10 @@ F: hmp-commands.hx
 T: git git://repo.or.cz/qemu/qmp-unstable.git queue/qmp
 
 Network device layer
-M: Stefan Hajnoczi <stefanha@redhat.com>
 M: Jason Wang <jasowang@redhat.com>
 S: Maintained
 F: net/
-T: git git://github.com/stefanha/qemu.git net
+T: git git://github.com/jasowang/qemu.git net
 
 Netmap network backend
 M: Luigi Rizzo <rizzo@iet.unipi.it>
